apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj

```diff
@@ -560,6 +562,24 @@
 			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 			showEnvVarsInLog = 0;
 		};
+		878250B85D4A451A7ACC3FB9 /* [CP] Embed Pods Frameworks */ = {
```

Comment:
All changes here seem to be unrelated to this PR 🤔 Looks like bare-expo pods were outdated on master — I've fixed it in https://github.com/expo/expo/commit/4b01ad7a81f241536824da61c224360fb6f40e30 so you can rebase it now